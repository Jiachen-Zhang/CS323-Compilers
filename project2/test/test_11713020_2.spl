int test_11713020_2()
{
  int a[3][1];
  a[3][0] = b;
  return 0;
}