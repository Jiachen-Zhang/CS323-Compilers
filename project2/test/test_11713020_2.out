Error type 1 at Line 4: undefined variable: b
Error type 5 at Line 4: unmatching type on both sides of assignment
